### PR TITLE
[PLT-0] Fix Integration Tests

### PR DESCRIPTION
**Pull request by adrian-chang**
# Description

* Add timestamps

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document change (fix typo or modifying any markdown files, code comments or anything in the examples folder only)

## Changes to Core Features

- [ ] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully run tests with your changes locally?
- [ ] Have you updated any code comments, as applicable?

### CODE REVIEW

File: .github/workflows/python-package-shared.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ jobs:
   integration:
     runs-on: ubuntu-latest
     concurrency:
-      group: labelbox-python-${{ inputs.test-env }}-${{ inputs.python-version }}
+      group: labelbox-python-${{ inputs.test-env }}-${{ inputs.python-version }}-integration
```
Comment (vbrodsky) [inline .github/workflows/python-package-shared.yml:39]: why is this added, curious

File: libs/labelbox/tests/data/annotation_import/test_bulk_import_request.py [hunk 2]
```diff
@@ -25,7 +25,7 @@
 
 """
 
-
+@pytest.mark.order(1)
```
Comment (vbrodsky) [inline libs/labelbox/tests/data/annotation_import/test_bulk_import_request.py:28]: and why is this?

File: pyproject.toml [hunk 3]
```diff
@@ -25,13 +25,17 @@ dev-dependencies = [
...
+    "pytest-order>=1.2.1",
 ]
 
 [tool.rye.workspace]
 members = ["libs/*", "examples"]
 
 [tool.pytest.ini_options]
-addopts = "-rP -vvv --reruns 1 --reruns-delay 5 --durations=20 -n auto --maxprocesses=10 --cov=labelbox --import-mode=importlib"
+# https://github.com/pytest-dev/pytest-rerunfailures/issues/99
+addopts = "-rP -vvv --reruns 1 --reruns-delay 5 --durations=20 -n 32 --cov=labelbox --import-mode=importlib --order-group-scope=module"
```
Comment (vbrodsky) [inline pyproject.toml]: any concerns increasing number of process? It seems that staging is overloaded as it is

Comment (vbrodsky) [reply pyproject.toml]: what did the option auto do

Comment (adrian-chang) [reply pyproject.toml]: Auto matches the cores on your machine, tbh I can revert this to auto then in CI use 32 to keep it simple (your laptop != ci machine in terms of compute)

File: libs/labelbox/tests/data/annotation_import/test_bulk_import_request.py (hunk 2 above)
Comment (adrian-chang) [reply libs/labelbox/tests/data/annotation_import/test_bulk_import_request.py:28]: This test is very slow, this guarantees it runs first (given all tests that you may be running) to prioritize it to optimize testing time

File: .github/workflows/python-package-shared.yml (hunk 1 above)
Comment (adrian-chang) [reply .github/workflows/python-package-shared.yml:39]: This ensures only 1 set of integration test is run per testing environment / python version

File: libs/labelbox/tests/data/annotation_import/test_data_types.py [hunk 4]
```diff
@@ -259,7 +198,7 @@ def validate_iso_format(date_string: str):
     assert parsed_t.minute is not None
     assert parsed_t.second is not None
 
-
+@pytest.mark.order(1)
```
Comment (adrian-chang) [inline libs/labelbox/tests/data/annotation_import/test_data_types.py:201]: These are to note to run first because they are slower

**Review by vbrodsky — APPROVED** [review]

**Review by martin-palermo — APPROVED** [review]
LGTM
